openage/convert/singlefile.py

```diff
@@ -297,45 +297,29 @@ def terrain_convert(drs, slp_path, palette_index, output_path, interfac=None):
 
     # import here to prevent that the __main__ depends on SLP
     # just by importing this singlefile.py.
-    from .slp import determine_rgba_matrix, SLP, SLPFrame, FrameInfo
+    from .slp import SLP, SLPFrame, FrameInfo
     from PIL import Image
-    from .texture import Texture, TextureImage
+    from .texture import Texture, TextureImage, merge_frames
     from .terrain_convert import Terrain_convert
     import numpy as np
```

Comment:
Few things here:

* imports for `SLPFrame` and `FrameInfo` are unncessary, since they are not referenced
* `Texture` is already imported at the start of the file. no need to reimport it here.

Running `make checkall` in the project root will find these redundancies for you (and also give code style hints).

Reply:
Yep you're right! I was not focused on the tests before making sure my code had at least a good logic. I'll correct that.